src/worker.js

```diff
@@ -36,8 +36,8 @@ function lintJob({ cliEngineOptions, contents, eslint, filePath }) {
     : cliEngine.executeOnFiles([filePath])
```

Comment:
Closest I can get to commenting while still be in the PR:

`contents` will _always_ be a string now, so this entire check is irrelevant now.